spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb

```diff
@@ -403,6 +403,7 @@ def assert_specific_vm_powered_on
     v.with_relationship_type("genealogy") do
       expect(v.parent).to eq(@template)
     end
+    expect(v.custom_attributes.find_by(:name => "Name").value).to eq("EmsRefreshSpec-PoweredOn-Basic3")
```

Comment:
would be nice if we could add 1 more tag to the testing data, but that can be done as follow up in the future, since it will require new VCRs